Statistics.Test/StatsUnitTest.cs

```diff
@@ -11,18 +12,18 @@ public void ReportsAverageMinMax()
         {
             var statsComputer = new StatsComputer();
             var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{1.5, 8.9, 3.2, 4.5});
+                new List<double>{1.5, 8.9, 3.2, 4.5});
             float epsilon = 0.001F;
-            Assert.True(Math.Abs(statsComputer.average - 4.525) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.max - 8.9) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.min - 1.5) <= epsilon);
+            Assert.True(Math.Abs(statsComputer.getAvg() - 4.525) <= epsilon);
+            Assert.True(Math.Abs(statsComputer.getMax() - 8.9) <= epsilon);
+            Assert.True(Math.Abs(statsComputer.getMin() - 1.5) <= epsilon);
         }
         [Fact]
         public void ReportsNaNForEmptyInput()
         {
             var statsComputer = new StatsComputer();
             var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{});
+                new List<double>{});
```

Comment:
This test will need to alert that empty input will give back NaN. Add this assert.